shared/api/storage.go

```diff
@@ -0,0 +1,37 @@
+package api
+
+// StoragePoolConfig represents the fields of a LXD storage pool.
+type StoragePool struct {
+	PoolName       string   `json:"name" yaml:"name"`
+	PoolDriver     string   `json:"driver" yaml:"driver"`
+	PoolUsedBy     []string `json:"used_by" yaml:"used_by"`
```

Comment:
Drop the "Pool" prefix there, it's a StoragePool struct :)

So "Name", "Driver" and "UsedBy".